<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>assignationWindow</class>
 <widget class="QMainWindow" name="assignationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assign</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="playerLabel">
        <property name="text">
         <string>Player</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="teamLabel">
        <property name="text">
         <string>Team</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="playerComboBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="teamComboBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="applyButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
